web/modules/custom/dept_migrate/modules/dept_migrate_group_nodes/config/install/migrate_plus.migration.group_node_easychart.yml

```diff
@@ -22,22 +22,23 @@ process:
       plugin: skip_on_empty
       method: row
       source: domain_node/0/target_id
-    -
-      plugin: skip_on_value
-      method: row
-      source: domain_node/0/target_id
-      value:
-        - 'daera'
-        - 'del'
-        - 'economy'
-        - 'execoffice'
-        - 'education'
-        - 'health'
-        - 'infrastructure'
-        - 'dcal'
-        - 'doe'
-        - 'justice'
-        - 'communities'
+#    -
+#      plugin: skip_on_value
+#      method: row
+#      source: domain_user/0/target_id
+#      # Whenever a site goes live (prod in D9), add its key to this list.
```

Comment:
The default state we'll begin with is to migrate everything for all sites. When a site launches to D9, prod content entry will happen there instead of D7 so we can begin to ignore things from that domain id from there on. The migrations will get faster as a result of having to process less material.